script/output/M2_deployment_data.json

```diff
@@ -1,9 +1,20 @@
 {
   "addresses": {
-    "beaconChainOracle": "0x90193C961A926261B756D1E5bb255e67ff9498A1"
```

Comment:
still not ignored

Reply:
copied the .gitignore line for M1 and realized it doesn't work lol

updated

should I fix the one for the M1 and remove that as well?

Reply:
ah, yes i guess. not sure if we're keeping them here for clarity? doubt it